### PR TITLE
Fixed the duplicated snapshot creation issue

### PR DESCRIPTION
**Pull request by lintongj**
1. Fixed the duplicated snapshot creation issue
2. Fixed the issue of parsing VC credential incorrectly 

Signed-off-by: Lintong Jiang <lintongj@vmware.com>

### CODE REVIEW

File: pkg/snapshotmgr/snapshot_manager.go [hunk 1]
```diff
@@ -114,6 +114,7 @@ func NewSnapshotManagerFromConfig(configInfo server.ConfigInfo, s3RepoParams map
 				logger.WithError(err).Errorf("Could not retrieve vsphere credential from k8s secret")
 				return nil, err
 			}
+			//configInfo.PEConfigs["ivd"] = ivdParams
```
Comment (lintongj) [inline pkg/snapshotmgr/snapshot_manager.go:117]: Will remove it.

File: pkg/snapshotmgr/snapshot_manager.go [hunk 2]
```diff
@@ -151,6 +152,7 @@ func NewSnapshotManagerFromConfig(configInfo server.ConfigInfo, s3RepoParams map
 	}
 
 	// Initialize the DirectProtectedEntityManager
+	logger.Infof("SnapshotManager: configInfo = %v", configInfo)
```
Comment (lintongj) [inline pkg/snapshotmgr/snapshot_manager.go:155]: Will remove it.

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/73/

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/74/

File: Makefile [hunk 3]
```diff
@@ -206,6 +206,9 @@ push: push-datamgr push-plugin push-backup-driver
 push-pp:
 	$(MAKE) push-plugin LOCALMODE=true VERSION=$(VERSION)-pp
 
+push-pp-wcp:
+	$(MAKE) push-plugin push-backup-driver LOCALMODE=true VERSION=$(VERSION)-pp-wcp
```
Comment (swgupta-vmw) [inline Makefile:210]: Should we just update this to the following so the the flag is set for data-manager as well?
push-pp:
	$(MAKE) push LOCALMODE=true VERSION=$(VERSION)-pp

Comment (lintongj) [reply Makefile:210]: We don't have to push data-manager image since PowerProtect don't use data-manager at all.

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/76/

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/79/

**Pull request merged**